tests/x11/kontact.pm

```diff
@@ -25,6 +25,11 @@ sub run() {
     sleep 2;
     x11_start_program("kontact", 6, {valid => 1});
 
+    # kontact has asking import data from another mailer
+    if (check_screen('kontact-import-data-dialog', 20)) {
```

Comment:
I know that 20s is long enough, but I really would like to leave default there. Is there any reason why use 20s instead of 30s? Looking at the code elsewhere it looks like we chose bad default since timeout is explicitely set almost everywhere (and even to 30 ;) ) and it's then copy&pasted all over the place.